src/pages/RestaurantDetail/Menus/Menu.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { MdKeyboardArrowDown } from 'react-icons/md';
+
+export default function Menu({ category_name, foods, id }) {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const toggleCollapsingMenu = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  useEffect(() => {
+    const isFirstCategory = id === 1;
+    if (isFirstCategory) {
+      setIsCollapsed(true);
+    }
```

Comment:
아래처럼 boolean값을 이용해 직접 작성 할 수도 있겠네요!
```suggestion
    const isFirstCategory = id === 1;
    setIsCollapsed(isFirstCategory);
```

Reply:
오호 감사합니다! 수정했습니다